web/src/beta/components/StorytellingPageSectionItem/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC } from "react";
+
+import { styled, useTheme } from "@reearth/services/theme";
+
+import Icon from "../Icon";
+import Text from "../Text";
+
+type Props = {
+  icon: string;
+  title: string;
+  index: number;
+  active?: boolean;
+  onAction?: () => void;
+  onClick?: () => void;
+};
+
+const StorytellingPageSectionItem: FC<Props> = ({
+  icon,
+  title,
+  index,
+  active,
+  onAction,
+  onClick,
+}) => {
+  const theme = useTheme();
+  return (
+    <HorizontalBox>
+      <VerticalBox>
+        <Text size={"body"}>{index}</Text>
+        <Icon icon={icon} />
+      </VerticalBox>
+      <TitleArea active={active}>
+        <Text
+          onClick={onClick}
+          size={"footnote"}
+          color={theme.general.content.strong}
+          otherProperties={{ wordBreak: "break-all" }}>
+          {title}
+        </Text>
+        <Icon icon="actionbutton" onClick={onAction} />
```

Comment:
I think the Icon needs a wrapper to avoid getting smaller due to long title text
<img width="460" alt="Screenshot 2023-06-22 at 11 09 21" src="https://github.com/reearth/reearth/assets/34051327/4e4bb967-8ac9-49bc-a90d-03931bd920b9">

Reply:
Fix completed.